xtask/src/count_loc.rs

```diff
@@ -0,0 +1,80 @@
+use std::{io::Write, path::PathBuf};
+
+use anstream::println;
+use color_eyre::{eyre::bail, Result};
+use glob::glob;
+use owo_colors::OwoColorize;
+use regex::Regex;
+use tempfile::NamedTempFile;
+use tokei::LanguageType;
+
+pub fn count_loc() -> Result<()> {
+    let mut results = Vec::new();
+    let config = tokei::Config::default();
+    let mut longest_path = 0;
+    for entry in glob("./*.rs")?.chain(glob("./src/**/*.rs")?) {
+        let path = entry?;
+        let tempfile = filter_lines(&path)?;
+
```

Comment:
I just looked at the tokei doc, looks like you can avoid writing to temp files by calling `LanguageType::Rust::parse_from_str(...)` to get the `CodeStats` directly

Reply:
d322c3c..1407de8